src/components/Summary/index.js

```diff
@@ -0,0 +1,63 @@
+import React from 'react';
+import { List } from 'antd';
+
+import { Container, ListItem, Profit, Footer, Content } from './styles.js';
+
+export default function Summary() {
+    // TODO: request from backend
+    const assets = [{
+        assetName: 'ITUB3',
+        averagePrice: 20,
+        sellPrice: 30,
+        totalSold: 10
+    }, {
+        assetName: 'ABV3',
+        averagePrice: 30,
+        sellPrice: 45,
+        totalSold: 20
+    }
+    ];
+
+    function calcProfit(asset) {
+        return (asset.sellPrice - asset.averagePrice) * asset.totalSold;
+    }
+
+    function totalProfit() {
+        return assets.reduce((a, b) => a + calcProfit(b), 0);
+    }
+
+    function calcTax() {
+        const profit = totalProfit();
+        return profit > 20000 ? profit * 0.15 : 0;
+    }
```

Comment:
Acho que essas regras de negócio poderiam vir tudo resolvido da API, e o frontend só se preocupar com apresentação e experiência. Podemos falar melhor depois!

Reply:
OK. Por enquanto posso adicionar na estrutura estática ali.

Reply:
I still left some methods there. But after we define our data structure I can change it.